install/streams.csv

```diff
@@ -1668,6 +1668,7 @@ RPCS,20,200,true,false,true,2002-06-24T00:00:00Z,9999-01-01T00:00:00Z
 RPZ,10,0.1,true,false,false,2001-06-06T00:00:00Z,9999-01-01T00:00:00Z
 RPZ,10,1,true,false,false,2001-06-06T00:00:00Z,9999-01-01T00:00:00Z
 RPZ,10,100,true,false,false,2001-06-06T00:00:00Z,9999-01-01T00:00:00Z
+RPZ,11,100,false,false,false,2020-05-01T07:34:00Z,9999-01-01T00:00:00Z
```

Comment:
Seems to be missing the 1 Hz and 0.1 Hz streams etc for the location 11.

Reply:
1 & 0.1 hz streams now added. Thanks @ozym !